src/PrestaShopBundle/Form/Admin/Sell/Product/SEO/SEOType.php

```diff
@@ -172,7 +174,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'label' => $this->trans('Tags', 'Admin.Catalog.Feature'),
                 'label_tag_name' => 'h2',
-                'help' => $this->trans('Use a comma to create separate tags. E.g.: dress, cotton, party dresses.', 'Admin.Catalog.Help'),
+                'help' => sprintf(
+                    '%s %s',
+                    $this->trans('To add tags, click in the field, write something, and then press the "Enter" key.', 'Admin.Shopparameters.Help'),
+                    $invalidCharsText
+                ),
```

Comment:
Shouldn't we update the translation message to include the invalid characters as a placeholder ?

Reply:
We could but it would include a new wording that needs to be translated in all languages by the crowdin community As much as possible we try to use the existing wording when available
But I'll test your proposal and we'll ask @Julievrz her opinion about this

Reply:
I agree for the paceholder Except this text is not a placeholder it's a help message under the input so it should be fine. I updated a few things to match the mockup. I didn't see that the tag components works with "Enter" AND comma key so I updated the old wording a bit to add both.

This is how it looks like now:

![Capture d’écran 2022-01-25 à 17 08 40](https://user-images.githubusercontent.com/13801017/151014083-714f77c5-d21e-45cf-a746-23e9653f20ae.png)

Both texts are maybe a bit redundant now so I'm open for suggestions :)